src/Pages/PopularRepositoriesTable/PopularRepositoriesTable.test.tsx

```diff
@@ -30,54 +30,66 @@ it('expect PopularRepositoriesTable to render with add one item', () => {
     data: { data: [defaultPopularRepository], meta: { count: 1, limit: 20, offset: 0 } },
```

Comment:
In this file, you have changed a lot of code that doesn't need to be change.
Revert all changes, and either: 

A. **Add** your own test

B. **Add** code to an existing test. 

Do not delete/move any code that currently exists here please.